openshift/console
# Add getting started section to the admin cluster dashboard

**Fixes**: 
Epic: https://issues.redhat.com/browse/ODC-5013
Story: https://issues.redhat.com/browse/ODC-5680

**Analysis / Root cause**: 
As an admin, I want to see the new getting started section with areas to "Set up your cluster" and other Learning resources on the Admin Overview page.

**Solution Description**: 
This PR is based on #8950 which adds a Getting started section to the Add page in the developer perspective. The first two commits should be reviewed as part of this commit.

This PR here should focus on the other two commits and the changes in the `public` package.

The new Getting started section in the admin perspective adds three content cards with different links:

1. "Set up your cluster" shows some good first steps to configure a cluster like "Add identity providers" and "Configure alert receivers". If an identity provider or the alert receivers are configured, these options disappear. If everything is configured the complete column disappears.
2. "Build with guided documentation" - This column shows up to two quick starts. It also prefers two hardcoded quickstarts (Monitoring your sample app and Quarkus with Helm). It but will automatically hide them when the admin completed them. When all quick started are completed, this column disappears.
3. "Explore new admin features" shows some static links to the API Explorer and a link to start customization.

**Screen shots / Gifs for design review**: 
@openshift/team-devconsole-ux

New getting started section:
![image](https://user-images.githubusercontent.com/139310/118573172-52570c00-b782-11eb-921a-7ecafd740dd3.png)

When the user wants to hide them:
![image](https://user-images.githubusercontent.com/139310/118573207-5edb6480-b782-11eb-8584-48e1ec7266ab.png)

When the page loads the first time and when the admin revisits the overview again (in one session):
![Peek 2021-05-18 02-41](https://user-images.githubusercontent.com/139310/118573385-ba0d5700-b782-11eb-94b6-01dd017aa53b.gif)

**Unit test coverage report**: 
![image](https://user-images.githubusercontent.com/139310/118573574-1bcdc100-b783-11eb-8698-41b7d28f0aa5.png)

**Test setup:**
* Open the admin perspective as kube:admin
* Check the links in the new getting started section
* Add an identity provider (click on the link, select Basic Auth and enter a URL like `http://localhost`)
* Configure an alert receiver (click on the link, select Configure for Critical and Default receivers, select Webhook and add an URL like `http://localhost`)

**Browser conformance**:
